program/Arnold II preliminary project proposal.md

```diff
@@ -0,0 +1,160 @@
+
+
+The project comprises two main components
+
+Component 1 - Design and Fabrication of the SoC Device
+
+Design and verification of the core 
+- assumed ready now
+
+FRONT end SoC design
+- eFPGA from Quicklogic (accelerator). Allows field upgradeable (commercial, military). Also wearable devices
+- Design of the SoC - based on Core-V MCU (updated Pulpissimo with last core) 
+-- key question: Peripheral connection model (uDMA vs AHB bus - does the OBI developed in CV32 which facilitates AHB connection need to be used)
+* Sep 21 keep uDMA
+
+-- shall we increase the SRAM (or MRAM for low power, density) or other compared to Arnold (512 kbyte)? Probably go to 1MByte. **What is the size/cost target of the device**
+* Sep 21 should we move to MRAM with 100MHz memory access (smaller but slower, non volatile) (ok for TensorFlow but not critical performance)?
+* Or dual memory architecture
+- Verification: intermediate level of quality, this is not a  
+--> assumed to be resourced primarily by QuickLogic
+
+Backend SoC design
+- leading to GDS2 netlist
+--> assumed to be resourced primarily by CMC
+- 
+- test insertion
+- on chip vs external body bias generator
+
+Fabrication
+- several thousand prototype
+- 22FDX Global Foundries
+- level of testing to be discussed
+
+
+Component 2- Design and Build of the Evaluation Kit
+- board 
+- SDK
+- customer can program the fpga to tune the accelerator
+
+
+
+### Summary of Timeline
+
+Fab submission in 2021 early part would be a preliminary goal. Market drivers - important to have devices in customers hands in mid 2021. Improvement vis a vis Arnold. Win-win between OpenHW (prove the part) and QuickLogic
+
+
+## OpenHW Members/Participants committed to participate in CORE-V IDE project
+
+CMC
+Quicklogic
+Global Foundries
+
+## Technical Project Leader(s) (TPLs)
+
+## Project Manager (PM)
+
+## Project Documents
+The following project documents will be created:
+
+
+
+## Summary of requirements
+
+* level of peripheral verification
+-- core should be same industrial quality
+-- what about peripherals such as I^2C - do they need the same level of verification
+
+
+* Put the key "product manager" requirements in this section. More detailed technology choices can be outsourced to a requirements document
+
+* fabrication requirements
+- timing, area --> what is important. E.g. to what extent is performance a key goal. TWG should weigh in.
+
+### Introduction
+
+### Initial project requirements 
+
+### Future enhancements:
+
+
+## Explanation of why OpenHW should do this project
+
+## Industry landscape: description of competing, alternative, or related efforts in the industry
+
+### Related efforts to be described
+
+## External dependencies
+- onchip bias from Global Foundries
+- MRAM memory? 
+- standard cells libraries and memory macros (Synopsis)
+- tools coming from Cadence
+- FLL or PLL: does this come from ETHZ? All involved: Global Foundries, Cadence, ETHZ

```

Comment:
Here we should also add the comments about it from yesterda, i.e. the idea of buying it externally from an IP vendor